build/crd-sync.sh

```diff
@@ -15,19 +15,33 @@ do
     || git clone -b main --depth 1 https://github.com/open-cluster-management-io/${REPO}.git .go/${REPO}
 done
 
+generate_v1beta1() {
+    CRD_PATH=${1}
+    yq '.apiVersion += "beta1"' -i ${CRD_PATH}
+    yq '.spec.version = "v1"' -i ${CRD_PATH}
+    yq '.spec.additionalPrinterColumns = .spec.versions[].additionalPrinterColumns' -i ${CRD_PATH}
+    yq '.spec.additionalPrinterColumns[] |= .JSONPath = .jsonPath' -i ${CRD_PATH}
+    yq 'del(.spec.additionalPrinterColumns[].jsonPath)' -i ${CRD_PATH}
+    yq '.spec.validation = .spec.versions[].schema' -i ${CRD_PATH}
+    yq '.spec.versions = [{"name": "v1", "served": true, "storage": true}]' -i ${CRD_PATH}
+    yq 'del(.. | select(has("default")).default)' -i ${CRD_PATH}
+    yq 'del(.. | select(has("oneOf")).oneOf)' -i ${CRD_PATH}
+    yq 'sort_keys(..)' -i ${CRD_PATH}
+}
```

Comment:
This is the `v1beta1` mutation. I mainly did it by observing the diff and adjusting accordingly, but also referenced: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122